lib/src/main/java/com/auth0/android/lock/internal/configuration/ApplicationFetcher.java

```diff
@@ -103,7 +102,7 @@ public void onResponse(Response response) {
                 }
 
                 Log.i(TAG, "Application received!");
-                callback.onSuccess(connections);
+                callback.onSuccess(new Configuration(connections, options));
```

Comment:
It's better to leave the responsibility of building the configuration to the caller